docs/report/report.tex

```diff
@@ -189,174 +189,164 @@ \subsection{Deployment Cycle}
+\subsubsection{Lösungen}
+Macio unterstützte das Team, indem sie einen ihrer Designer bereitstellten.
+Dieser half, die bestehenden Mockups zu verbessern.
+Das Erstellen von und Auseinandersetzen mit Sequenzdiagrammen half bei dem Team dabei, besser das Verhalten von einem Nutzer einzuschätzen und entsprechend das User Interface zu konzipieren.
+\subsubsection{Product Increment}
+Zusätzlich zu dem Wunsch, dass beim anlegen eines NFC-Tags auch die jeweilig hinterlegte Spotify Musik abgespielt wurde, war auch das Einloggen über einen bestehenden Spotify Account möglich.
+Auf der Startseite wurden auch die NFC-Tags entsprechend der zugeordneten Gruppe gruppiert.
+Jegliche Neuerung auf dem Master wurden nun auch auf Github.io veröffentlicht.
+\subsubsection{Retrospektive}
+Deutlich besser lief in diesem Sprint die Problembewältigung durch Pair Programming, welches weiter die Teamarbeit und gegenseitige Hilfe verbesserte.
+Auch der Workflow mit Github, den Pull Request sowie Code Reviews wurde mehr verinnerlicht.
+Das Team lernte in diesem Sprint vor allem Vue Hooks kennen, und die Stärken von Feathers in Verbindung mit OAuth.
+Weiterhin waren die Tickets stärker voneinander abhängig, weshalb manche Arbeiten nicht direkt möglich machte.
+Um zukünftige Probleme gar nicht erst entstehen zu lassen, hatte sich das Team vorgenommen, entstandene Probleme nicht erst im Standup, sondern früher zu kommunizieren.
+Genauso sollte neu Gelerntes besser kommuniziert werden, um die Arbeit effizienter zu gestalten.
+
+\subsection{Meilenstein 4}
+Meilenstein vier lag innerhalb der Weihnachtszeit und wurde deswegen mit einem größeren Zeitraum vom 17.12.2020 bis 14.01.2021 geplant.
+Mit diesem Meilenstein begann die funktionale und visuelle Umsetzung der Hauptfunktionen.
+\subsubsection{Ziel}
+Wie auch beim vorherigen Meilenstein war wieder hier der Fokus auf den Kundenwunsch, dass die ersten Funktionalitäten umgesetzt werden.
+Dass umfasste das Anlegen von eigenen Tags, also dem Scannen von Tags, vergeben eines eigenen Namens, auswählen von Musik und einem eigenen Bild, sowie das bearbeiten bereits angelegter Tags.
+Auch sollten für den nächsten Schritt die Mockups erstellt werden, wie ein Nutzer den Lautsprecher zur Wiedergabe auswählt sowie seine eigene \textit{Leek Box} initial einrichtet.
+\subsubsection{Probleme}
+Zu diesem Meilenstein sind vermehrt größere Probleme aufgetreten.
+Die Planung und Einschätzung der umsetzbaren Tickets wurde schlecht eingeschätzt.
+Es wurde sich zu sehr in Details verloren, sodass der Fokus auf die wichtigen Tickets verloren ging.
+Das führte dazu, dass am Ende viele Tickets halbfertig und/oder nur schlecht getestet implementiert waren.
+\subsubsection{Lösungen}
+Kurzfristig wurde eine Team interne Evaluation der einzelnen Mitglieder durchgeführt.
+Das half einerseits dabei, dass Probleme gelöst und die Teammotivation sowie die Produktivität stieg, als auch das generelle Teamklima weiter gebessert wurde.
```

Comment:
Hier auf jeden Fall noch anführen, was wir als Lösung für die halbfertigen Tickets gewählt haben.
So klingt es sehr stark danach, dass die Lösung die Evaluation war. So war es doch nicht, oder?

Reply:
Da ich im Satz davor schon geschrieben hab, dass halbfertige tickets entweder schelcht getestet oder gar nicht drin sind, finde ich muss man das nicht nochmal in lösung schreiben

Reply:
```suggestion
Kurzfristig wurde eine Team interne Evaluation der einzelnen Mitglieder durchgeführt.
Das half einerseits dabei, dass Probleme gelöst und die Teammotivation sowie die Produktivität stieg, als auch das generelle Teamklima weiter gebessert wurde.
Darüber hinaus wurde auch beschlossen, in den folgenden Sprints die Menge an zu bearbeitenden Issues besser festzulegen. 
```